.travis.yml

```diff
@@ -12,10 +12,19 @@ env:
     - CXX: g++-4.9
     - CC: gcc-4.9
 
-node_js:
-  - '9'
-  - '8'
-  - '7'
-  - '6'
-  - '5'
-  - '4'
+matrix:
+  include:
+  - node_js: '9'
+    env: [NSP=true]
+  - node_js: '8'
+    env: [NSP=true]
+  - node_js: '7'
+    env: [NSP=true]
+  - node_js: '6'
+    env: [NSP=true]
+  - node_js: '5'
+  - node_js: '4'
+
+script:
+  - npm run test
+  - test $NSP && npm run audit || echo "skip audit"
```

Comment:
Nice one!

Reply:
Hm wait. If `npm run audit` fails, does it fail the build?

Reply:
@vweevers Ideally it should, right? Looks like operator precedence might short circuit this check.

Reply:
```
$ test $nope && test $nope || echo "oops"
oops
$ echo $?
0
```

To ensure the exit code of `npm run audit` is the exit code of the line, we can do:

```
if [ "$NSP" ]; then npm run audit; else echo "skip audit"; fi
```

Reply:
#198